### ISSUE TITLE
Welcome message from React and DirectLine

### ISSUE THREAD

**Issue by vglu** [microsoft/BotFramework-WebChat#3187]
Hi

In provided example Bot not says welcome hello. If we are run it with QnA bot - we can see lame behavior. We have hot first message from bot only after send our first message.

https://microsoft.github.io/BotFramework-WebChat/01.getting-started/g.hybrid-react-npm/

Expected behavior - when we open page we should get welcome message from bot.

Browser - Chrome

Thanks

Vitaliy

**Comment by compulim** [comment]
This doc should answers to your question about the first message, https://github.com/microsoft/BotFramework-WebChat/blob/master/docs/WELCOME_MESSAGE.md.

Please open a new issue if it don't, we will address the doc accordingly.
